### ISSUE TITLE
Any way to help me to make it work again?

### ISSUE THREAD

**Issue by Casal0x** [rastaunlimited/Helbreath#1]
I'm trying to make it work again this files and start developing features until it stay in the same state of olympia. Cloud you help me with the update of dependencies so i can start in a working code?

**Comment by khalil2099** [comment]
hey, are you still developing helbreath game ? im just starting.

**Comment by Casal0x** [comment]
yes but I writed my own files now never got an answer and with other guys developed a new client that works 10 times better, the first version of it, its the one used in helbreath war (its an older one and i'm not the owner of that server).

**Comment by khalil2099** [comment]
are you running any servers now ? or just helping helbreath war ?

**Comment by Casal0x** [comment]
Not yet and not helping helbreath war xD he made a mess with ia. so just ending my files until they are in the best state posible

**Comment by khalil2099** [comment]
since you said you wrote your own files, is it still c++ or you rewrite everything from scratch ? right now im not sure wether to upgrade the c++ source code or just rewrite from scratch but use the same assets.
